geoapps/plotting/plotting.py

```diff
@@ -230,8 +230,16 @@ def plot_plan_data_selection(entity, data, **kwargs):
             if not np.any(entity.get_data(key)):
                 continue
 
+            line_data = entity.get_data(key)[0]
+            if isinstance(line_data, ReferencedData):
+                values = [
+                    key
+                    for key, value in line_data.value_map.map.items()
+                    if value in values
+                ]
```

Comment:
**question:** not sure to understand why we need filtering if ReferenceData but not in the other cases

Reply:
Want to expose the string values to the users, but do the logical indexing based on the keys (integer). I agree that a get_key from value would make sense.

I would suggest we leave it as is for now, create the function on geoh5py, and clean this up here after?

Reply:
yep, could create a technical story in JIRA ;)
so that when this is done in geoh5py, it is remembered to update the code here